tests/MyGame/Example/ArrayStruct.py

```diff
@@ -49,3 +87,62 @@ def CreateArrayStruct(builder, a, b, c, d_a, d_b, d_c, d_d, e, f):
         builder.PrependInt32(b[_idx0-1])
     builder.PrependFloat32(a)
     return builder.Offset()
+
+import MyGame.Example.NestedStruct
+try:
+    from typing import List
+except ImportError as error:
+    print(error.__class__.__name__ + ": " + error.message)
```

Comment:
Will this print for every Python2 user? We know typing doesn't work on Python2 so it seems superfluous.

Reply:
Good point! I've replaced it with pass.